src/App.jsx

```diff
@@ -1,25 +1,20 @@
-import { BrowserRouter } from 'react-router-dom';
+import { Switch, BrowserRouter, Route } from 'react-router-dom';
 import React from 'react';
-import styled from 'styled-components';
-import NavBar from './NavBar';
-
-// Styling section
-
-const Container = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 100%;
-  background-color: #383854;
-`;
+import Home from './Home';
+import Calculator from './Calculator';
+import Prices from './Prices';
+import Statistics from './Statistics';
 
 // our App
 
 const App = () => (
   <BrowserRouter>
-    <Container>
-      <NavBar />
-    </Container>
+    <Switch>
```

Comment:
You have the NavBar in all pages. A nice way to do this
```js
const App = () => (
  <BrowserRouter>
    <div>
      <NavBar />
      <Switch>
        <Route exact path="/" component={Home} />
        <Route path="/calculator" component={Calculator} />
        <Route path="/prices" component={Prices} />
        <Route path="/statistics" component={Statistics} />
      </Switch>
    </div>
  </BrowserRouter>
);
```
Then there is no need to import NavBar into every component